modules/primer-buttons/README.md

```diff
@@ -245,6 +245,21 @@ Use `.hidden-text-expander` to indicate and toggle hidden text.
 
 You can also make the expander appear inline by adding `.inline`.
 
+
+#### `<details>`/`<summary>` button styles
```

Comment:
I don't think we should use code snippets in the title, that will be a bit odd when rendered in the `toc` and will look odd in the title.